composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require":      {
-        "php": ">=7.1",
+        "php": ">=7.4",
```

Comment:
Why is this needed? All required typehints can be added in php 7.1+

Reply:
You're right, switched back in a0f032098f7eb02e53e5de21b76a28e42cb4333a.
Sorry.